### PR TITLE
[docs] Update README.md

### PR DESCRIPTION
**Pull request by cuiyanx**
Signed-off-by: Cui, Yan yanx.cui@intel.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ $ sudo usermod -a -G plugdev USERNAME
 ```
 
 #### Add udev rules
-Copy these two files into your /etc/udev.rules directory:
+Copy these two files into your /etc/udev/rules.d directory:
```
Comment (kenchris) [inline README.md]: I have both directories on ubuntu, do you know the difference?

Comment (cuiyanx) [inline README.md]: If setup, there are only one director on Ubuntu 16.04 host; If update, you have both.

Comment (grgustaf) [inline README.md]: How about change like this then:

... /etc/udev/rules.d directory (/etc/udev.rules for Ubuntu 14.04):

Comment (cuiyanx) [inline README.md]: Great！

**Comment by jimmy-huang** [comment]
+1

**Pull request merged**